main.cpp

```diff
@@ -72,45 +85,82 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 } 
 
 /*
- 1)
+ 1)Rent a bike
  */
-
+bool rentBike(double rentDate,int amountOfMoney= 10)
+{
+  ignoreUnused(rentDate,amountOfMoney);
+  return{};
+}
```

Comment:
Check the coding standard in the Readme.md for how to format your code.
also see the comments above regarding spacing around the `=` and `,`
- [ ] revise